src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -104,19 +111,19 @@ function getCurveInputs(modelInputs: EpidemicModelState[]) {
   });
 }
 
-function getHospitalBeds(modelInputs: EpidemicModelState[]) {
-  let sumHospitalBeds = 0;
-  modelInputs.forEach((input) => {
-    return (sumHospitalBeds += input.hospitalBeds || 0);
-  });
-  return sumHospitalBeds;
-}
-
 const ResponseImpactDashboard: React.FC = () => {
   const { data: localeDataSource } = useLocaleDataState();
   const [scenarioState] = useScenario();
   const [curveInputs, setCurveInputs] = useState([] as CurveFunctionInputs[]);
+  const [originalCurveInputs, setOriginalCurveInputs] = useState(
```

Comment:
If we call these `originalCurveInputs` and `originalWhatever`, let's also rename the variables for the current projection to `currentCurveInputs` and `currentWhatever` to match.